recipes/cvplot/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CvPlotConan(ConanFile):
+    name = "cvplot"
+    description = "fast modular opencv plotting library"
+    license = "MIT License"
+    topics = ("plot", "opencv")
```

Comment:
```suggestion
    topics = ("plot", "opencv", "diagram", "plotting")
```

From  upstream github's page